modules/codegen/src/main/scala/com/twilio/guardrail/SwaggerUtil.scala

```
@@ -550,8 +550,13 @@ object SwaggerUtil {
                   case t"Int"        => Right(p"IntVar(${Pat.Var(paramName)})")
                   case t"Long"       => Right(p"LongVar(${Pat.Var(paramName)})")
                   case t"BigInt"     => Right(p"BigIntVar(${Pat.Var(paramName)})")
-                  case tpe @ Type.Name(_) =>
+                  case Type.Name(tpe) =>
                     Right(p"${Term.Name(s"${tpe}Var")}(${Pat.Var(paramName)})")
+                  case Type.Select(_, Type.Name(tpe)) =>
+                    Right(p"${Term.Name(s"${tpe}Var")}(${Pat.Var(paramName)})")
+                  case tpe =>
+                    println(s"Error: Unsure how to map ${tpe} into an extractor")
```

This is necessary because atto eats the error string before we get here